hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/client/functional/TestConsistentBucketIndex.java

```diff
@@ -255,7 +256,7 @@ private int readRecordsNum(String[] partitions, boolean populateMetaFields) {
    */
   private List<WriteStatus> writeData(String commitTime, int totalRecords, boolean doCommit) {
     List<HoodieRecord> records = dataGen.generateInserts(commitTime, totalRecords);
-    JavaRDD<HoodieRecord> writeRecords = jsc.parallelize(records, 2);
+    JavaRDD<HoodieRecord> writeRecords = jsc.parallelize(records, 1);
```

Comment:
why change the parallelism?

Reply:
is it because the metadata file creation conflicts?

Reply:
> why change the parallelism?

To pass ut. 

> is it because the metadata file creation conflicts?

Yes, Because local fs does not create temporary files, it accesses the intermediate state when concurrently reading/creating metadata files.